docs/tuning-and-overhead.asciidoc

```diff
@@ -120,6 +120,8 @@ we've tagged certain configuration options in the documentation with _(performan
 ==== Sample rate
 
 _Sample rate_ is the percentage of requests which should be recorded and sent to the APM Server.
+(For pre-8.0 servers, unsampled requests are sent without contextual information which reduces
```

Comment:
Something I had to dig for after reading this is: what about metrics?
E.g. extending the explanation a little here to show the implications might be helpful:
```
from 8.0 unsampled requests are not sent at all, request metrics are extrapolated on the server side based on the sampling rate.
```

If you feel like this doesn't belong here you can keep it as is.